types/schemas/app/ApplicationData.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ApplicationTypeKeys,
+  PrimaryApplicationType,
+} from '../digitalPlanningApplication/enums/ApplicationTypes';
+
+/**
+ * @internal
```

Comment:
`@internal` is required to not expose generic types when building the JSON schema, and to force `ts-json-schema-generator` to roll this up into its nearest parent type.

Generated JSON schema without the `@internal` notation -

<img width="1362" alt="image" src="https://github.com/user-attachments/assets/43cc436f-c691-4376-9fd2-483de07f882f">

<img width="1360" alt="image" src="https://github.com/user-attachments/assets/4af4ddca-a2a4-47e4-8977-4394ca431c37">